src/rat/plugins/forecasting.py

```diff
@@ -379,6 +744,128 @@ def generate_forecast_state_and_inputs(
     return state_outpath, forcings_outpath
 
 
+def convert_forecast_inflow(inflow_dir, reservoir_shpfile, reservoir_shpfile_column_dict,  final_out_inflow_dir, basedate):
+    # Inflow
+    reservoirs = gpd.read_file(reservoir_shpfile)
+    reservoirs['Inflow_filename'] = reservoirs[reservoir_shpfile_column_dict['unique_identifier']].astype(str)
+
+    inflow_paths = list(Path(inflow_dir).glob('*.csv'))
+    final_out_inflow_dir.mkdir(exist_ok=True)
+
+    for inflow_path in inflow_paths:
+        res_name = os.path.splitext(os.path.split(inflow_path)[-1])[0]
+
+        if res_name in reservoirs['Inflow_filename'].tolist():
+            savepath = final_out_inflow_dir / inflow_path.name
+
+            df = pd.read_csv(inflow_path, parse_dates=['date'])
+            df['inflow (m3/d)'] = df['streamflow'] * (24*60*60)        # indicate units, convert from m3/s to m3/d
+            df = df[['date', 'inflow (m3/d)']]
+
+            print(f"Converting [Inflow]: {res_name}")
+            df = df[df['date'] > basedate]
+            df.to_csv(savepath, index=False)
+            print(df.tail())
+        else:
+            print(f"Currently not displayed in website: {res_name}")
```

Comment:
We should not use website. Rather may be something like: Skipping {res_name} as its inflow file is not available.